docs/cvl/methods.md

```diff
@@ -178,6 +190,63 @@ shown in the web report can indicate whether a summary was applied.
 
 ```
 
+(catch-unresolved-calls-entry)=
+### Catch unresolved-calls entry
+The catch unresolved-calls entry is a special type of summary declaration that instructs the Prover to replace calls to unresolved external function calls with a specific kind of summary, dispatch list.
+By default, the Prover will use an AUTO summary for unresolved function calls, but that may produce spurious counter examples. 
+The catch unresolved-calls entry let's the user refine the summary used for unresolved function calls.
+
+```{note}
+Only one catch unresolved-calls entry is allowed per a specification file```
```

Comment:
What happens if we import and try to override? All answers correct, just document it

Reply:
I believe we don't have override for methods block entries, but added text saying it gets imported, and cannot be overriden.